File: front/src/pages/Jydboot/SysLog/SysLog.js
import React, { PureComponent } from 'react';
import { connect } from 'dva';
import { Button, Col, Form, Input, Row } from 'antd';
import Panel from '../../../components/Panel';
import { SYSLOG_LIST } from '../../../actions/sysLog';
import Grid from '../../../components/Sword/Grid';

const FormItem = Form.Item;

@connect(({ sysLog, loading }) => ({
  sysLog,
  loading: loading.models.sysLog,
}))
@Form.create()
class SysLog extends PureComponent {
  // ============ 查询 ===============
  handleSearch = params => {
    const { dispatch } = this.props;
    dispatch(SYSLOG_LIST(params));
  };

  // ============ 查询表单 ===============
  renderSearchForm = onReset => {
    const { form } = this.props;
    const { getFieldDecorator } = form;

    return (
      <Row gutter={{ md: 8, lg: 24, xl: 48 }}>
        <Col md={6} sm={24}>
          <FormItem label="查询名称">
            {getFieldDecorator('name')(<Input placeholder="查询名称" />)}
          </FormItem>
        </Col>
        <Col>
          <div style={{ float: 'right' }}>
            <Button type="primary" htmlType="submit">
              查询
            </Button>
            <Button style={{ marginLeft: 8 }} onClick={onReset}>
              重置
            </Button>
          </div>
        </Col>
      </Row>
    );
  };

  render() {
    const code = 'sysLog';

    const {
      form,
      loading,
      sysLog: { data },
    } = this.props;

    const columns = [
      {
        title: '主键',
        dataIndex: 'id',
      },
      {
        title: '服务器IP地址',
        dataIndex: 'serverIp',
      },
      {
        title: '操作地址ip',
        dataIndex: 'remoteIp',
      },
      {
        title: '服务名称',
        dataIndex: 'serviceName',
      },
      {
        title: '系统模块',
        dataIndex: 'optModule',
      },
      {
        title: '操作类型',
        dataIndex: 'optType',
      },
      {
        title: '日志类型1管理端2门户3移动端',
        dataIndex: 'type',
      },
      {
        title: '请求方法',
        dataIndex: 'method',
      },
      {
        title: '请求路径',
        dataIndex: 'requestUri',
      },
      {
        title: '返回结果',
        dataIndex: 'response',
      },
      {
        title: '方法名',
        dataIndex: 'methodName',
      },
      {
        title: '方法类',
        dataIndex: 'methodClass',
      },
      {
        title: '操作提交的数据',
        dataIndex: 'params',
      },
      {
        title: '操作状态0成功1失败',
        dataIndex: 'status',
      },
      {
        title: '创建者',
        dataIndex: 'createUser',
      },
      {
        title: '创建时间',
        dataIndex: 'createTime',
      },
    ];

    return (
      <Panel>
        <Grid
          code={code}
          form={form}
          onSearch={this.handleSearch}
          renderSearchForm={this.renderSearchForm}
          loading={loading}
          data={data}
          columns={columns}
        />
      </Panel>
    );
  }
}
export default SysLog;

File: src/main/java/com/jyd/entity/SysLog.java
package com.jyd.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import java.util.Date;
import java.io.Serializable;
import lombok.Data;
import lombok.EqualsAndHashCode;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * 系统日志实体类
 *
 * @author 
 * @since 2023-12-01
 */
@Data
@TableName("t_sys_log")
@ApiModel(value = "SysLog对象", description = "系统日志")
public class SysLog implements Serializable {

	private static final long serialVersionUID = 1L;

	/**
	* 主键
	*/
		@ApiModelProperty(value = "主键")
		private String id;
	/**
	* 服务器IP地址
	*/
		@ApiModelProperty(value = "服务器IP地址")
		private String serverIp;
	/**
	* 操作地址ip
	*/
		@ApiModelProperty(value = "操作地址ip")
		private String remoteIp;
	/**
	* 服务名称
	*/
		@ApiModelProperty(value = "服务名称")
		private String serviceName;
	/**
	* 系统模块
	*/
		@ApiModelProperty(value = "系统模块")
		private String optModule;
	/**
	* 操作类型
	*/
		@ApiModelProperty(value = "操作类型")
		private String optType;
	/**
	* 日志类型1管理端2门户3移动端
	*/
		@ApiModelProperty(value = "日志类型1管理端2门户3移动端")
		private String type;
	/**
	* 请求方法
	*/
		@ApiModelProperty(value = "请求方法")
		private String method;
	/**
	* 请求路径
	*/
		@ApiModelProperty(value = "请求路径")
		private String requestUri;
	/**
	* 返回结果
	*/
		@ApiModelProperty(value = "返回结果")
		private String response;
	/**
	* 方法名
	*/
		@ApiModelProperty(value = "方法名")
		private String methodName;
	/**
	* 方法类
	*/
		@ApiModelProperty(value = "方法类")
		private String methodClass;
	/**
	* 操作提交的数据
	*/
		@ApiModelProperty(value = "操作提交的数据")
		private String params;
	/**
	* 操作状态0成功1失败
	*/
		@ApiModelProperty(value = "操作状态0成功1失败")
		private Integer status;
	/**
	* 创建者
	*/
		@ApiModelProperty(value = "创建者")
		private String createUser;
	/**
	* 创建时间
	*/
		@ApiModelProperty(value = "创建时间")
		private Date createTime;


}
